app/views/shipments/_shipping_table.html.erb

```diff
@@ -4,7 +4,6 @@
       <% if local_assigns[:show_order_number] %>
         <th>Order Number</th>
       <% end %>
-      <th>Shipping Date</th>
       <th>Tracking Number</th>
       <th>Carrier</th>
       <th>Delivery Date</th>
```

Comment:
Didn't we want to remove the Delivery date as well?  (Until we hook up the tracking API stuff.)